docs/pages/identity-security/session-summaries.mdx

```diff
@@ -0,0 +1,339 @@
+   metadata:
+     name: shell-summary-model
+   spec:
+     # "openai" is currently the only supported inference provider.
+     openai:
+       # In this scenario, where we connect to OpenAI directly, openai_model_id
+       # needs to be a valid OpenAI model name.
+       openai_model_id: gpt-4o
+       # This value refers to the `inference_secret` resource name.
+       api_key_secret_ref: openai-key
+     # It's a good idea to limit the session length to prevent incurring cost for
+     # sessions that were too long anyway to fit in the model's context window.
+     # If unset, defaults to 200kB. See the Limitations section for more details.
+     max_session_length_bytes: 190000
+
+   ---
+
+   kind: inference_secret
+   version: v1
+   metadata:
+     name: openai-key
+   spec:
+     value: '<paste-your-openai-api-key-here>'
+
+   ---
+
+   kind: inference_policy
+   version: v1
+   metadata:
+     name: shell-sessions
+     description: Summarize all sessions using OpenAI
+   spec:
+     # Session kinds affected by this policy. Allowed values: "ssh", "k8s", "db".
+     kinds:
+     - ssh
+     - k8s
+     - db
+     # Name of the `inference_model` resource that will be used for summarizing
+     # these sessions.
+     model: shell-summary-model
+   ```
+
+1. Create the configuration resources in the backend:
+
+   ```code
+   $ tctl create -f inference-config.yaml
+   ```
+
+For full description of the inference configuration resources, see appropriate
+sections of [Teleport Resources Reference](../reference/resources.mdx):
+
+- [`inference_model`](../reference/resources.mdx#inference-model)
+- [`inference_secret`](../reference/resources.mdx#inference-secret)
+- [`inference_policy`](../reference/resources.mdx#inference-policy)
+
+## Step 3/4: Configure access to session summaries.
+
+Every user who has access to
+[given session recording](../reference/access-controls/roles.mdx#rbac-for-session-recordings)
+also has access to its summary through a `session` resource kind. The built-in
+`access` role allows users to only view their sessions; in order to allow a
+user to view all recordings and summaries, it's easiest to use the built-in
+`auditor` role.
+
+You can either use the built-in roles or create a new one to make sure you have
+access to all recordings. To create a new role:
+
+1. Create a `summary-auditor.yaml` file with the following content:
+
+   ```yaml
+   kind: role
+   metadata:
+     name: summary-auditor
+   spec:
+     allow:
+       rules:
+       - resources:
+         - session
+         verbs:
+         - read
+         - list
+   version: v7
+   ```
+
+1. Apply your changes:
+
+   ```code
+   $ tctl create -f summary-auditor
+   ```
+
+1. (!docs/pages/includes/add-role-to-user.mdx role="summary-auditor"!)
+
+
+## Step 4/4: Conduct a session and view its summary
+
+That's it! We are ready to conduct a test session. Connect to any of your
+cluster resources using `tsh ssh`, `kubectl`, `tsh db connect`, or a web
```

Comment:
```suggestion
cluster resources using `tsh ssh`, `kubectl exec`, `tsh db connect`, or a web
```

Would `tsh db connect` actually trigger a summary? My understanding is that this will only work for Postgres (and soon to be MySQL) sessions run from the Web UI.